### DISCUSSION TITLE
Beginners Question ERROR ID : 100103

### DISCUSSION THREAD

**Discussion by rmoortgat** [OpenRadioss/OpenRadioss#2265] [Q&A]
Hello,

I tried the TubeÏmpact example.
I get the following error:
ERROR ID : 100103
** ERROR IN INPUT OPTIONS
DESCRIPTION :  
-- BLOCK: /GRNOD/NODE/1
-- LINE: -    node IDS    --
Cannot read an integer value
SOLUTION :     
Please check input format and fields description defined in the Reference Guide.

I checked the reference guide. I suppose it has something to do with the fields. I rearranged the values a bit, but to no avail.
I wrote the input file with notedpad++.

Thanks for your help.
[Tube_Impact_0000.txt](https://github.com/OpenRadioss/OpenRadioss/files/14022377/Tube_Impact_0000.txt)

Roger

**Comment by MarianBulla** [comment]
Hello Roger,
and thanks for your interest in OpenRadioss.
Regarding your ERROR: There are some bad formatted lines in your input-dec:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/bba8a644-5fbe-4c61-b7ed-c69a271162a9)

The main issue is in line 11347. But the others should be corrected, too.
Like line #11308, 11323, 11334, 11344 and so on.

Also one suggestion: In your property card, since you are using an elasto-plastic material model, set Ithick and Iplast to 1.

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by rmoortgat** [comment]
Thank you!
It works now.


met vriendelijke groeten | bien à vous | kind regards

[Logo Establis]<http://www.establis.eu/>
ROGER MOORTGAT
SR EXPERT STABILITEIT
Roeselare - Antwerpen - Gent
follow us: [linkedIn] <https://www.linkedin.com/company/establis> [Faceboook] <https://www.facebook.com/establisgroup/> [Instagram] <https://www.instagram.com/establis_group/>

disclaimer<http://www.establis.eu/en/disclaimer> - privacyverklaring<http://www.establis.eu/nl/privacyverklaring>

**Reply by MarianBulla** [reply]
This is great !
Love to hear that :-) 

Thanks for your info and have fun with your further simulations.

Thanks and best regards,
Marian
